src/communication/CommandLine.cpp

```diff
@@ -205,7 +204,7 @@ void CommandLine::sliceNext()
 
                         const FMatrix3x3 transformation = last_settings->get<FMatrix3x3>("mesh_rotation_matrix"); //The transformation applied to the model when loaded.
 
-                        if (!loadMeshIntoMeshGroup(&slice.scene.mesh_groups[mesh_group_index], argument.c_str(), transformation, last_extruder.settings))
+                        if (!loadMeshIntoMeshGroup(&slice.scene.mesh_groups[mesh_group_index], argument.c_str(), transformation, *last_settings))
```

Comment:
I think, if you're slicing two models with per-object settings, this would result in the settings from the first object bleeding over into the second object. Like this:

    CuraEngine -l first.stl -s infill_line_distance=1.2 -l second.stl

`second.stl` would also get sliced with 1.2mm infill line distance, right?